packages/hub/src/graphql/queries/runSquiggle.ts

```diff
@@ -82,6 +82,7 @@ export async function runSquiggle(code: string): Promise<SquiggleOutput> {
   const env = {
     sampleCount: 1000, // int
     xyPointLength: 1000, // int
+    seed: "default_seed",
```

Comment:
Could be in GraphQL args, but not important for now.

Reply:
Yea, I decided not to do all of that in this story. This story was supposed to mainly be about the UI